### PR TITLE
Removed tailwind class from pages component part 1

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Worked on removing tailwind for page level component part 1

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/CustomPropertiesPageV1/CustomPropertiesPageV1.less [hunk 1]
```diff
@@ -16,3 +16,8 @@
     padding-top: 0px;
   }
 }
+
+.customPropertiesSchemaEditor {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/CustomPropertiesPageV1/CustomPropertiesPageV1.less:20]: class name should be `kabab-case`

File: openmetadata-ui/src/main/resources/ui/src/pages/EditConnectionFormPage/EditConnectionFormPage.component.tsx [hunk 2]
```diff
@@ -150,14 +151,14 @@ function EditConnectionFormPage() {
       </ErrorPlaceHolder>
     ) : (
       <PageLayout
-        classes="tw-max-w-full-hd tw-h-full tw-pt-4"
+        classes="max-w-full-1080 h-full p-t-xss"
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/EditConnectionFormPage/EditConnectionFormPage.component.tsx:154]: `max-w-full-1080` full  & 1080 both? what does it mean

File: openmetadata-ui/src/main/resources/ui/src/pages/login/LoginCarousel.tsx [hunk 3]
```diff
@@ -28,15 +28,16 @@ const LoginCarousel = () => {
...
               />
             </div>
-            <div className="tw-mt-24 tw-mb-11">
+            <div className="mt-24 mb-11">
               <p
-                className="tw-text-center tw-w-5/6 tw-mx-auto tw-font-medium tw-text-white tw-text-xl"
-                data-testid="carousel-slide-description">
+                className="text-center mx-auto font-medium text-white text-xl"
+                data-testid="carousel-slide-description"
+                style={{ width: '83%' }}>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/login/LoginCarousel.tsx:40]: instead width 83% you can give marginX right?

**Comment by ShaileshParmar11** [comment]
Hi @chirag-madlani I have push the changes!

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**